src/runtime/inertia/components/Link.vue

```diff
@@ -110,72 +110,53 @@ const linkClass = computed(() => {
 })
 
 const page = usePage()
-const url = computed(() => props.to ?? props.href ?? '#')
+const url = computed(() => props.to ?? props.href ?? '')
 
 const isActive = computed(() => props.active || (props.exact ? url.value === props.href : page?.url.startsWith(url.value)))
```

Comment:
```suggestion
const isActive = computed(() => props.active || (!!url.value && (props.exact ? url.value === props.href : page?.url.startsWith(url.value))))
```
This fixes an issue where `isActive` would always be true if the `url` is not defined.